server/api/categoryRouter.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Router for item actions
+ */
```

Comment:
Add a todo to add testing

Reply:
Should these be official github issues?

Reply:
sure, about github issues- we could? but the TODOs for testing would be in the category test file tho right?